generated/ridp/user_exists_in_enroll_SHOP.json

```diff
@@ -0,0 +1,29 @@
+{
+  "primary_applicant": {
+    "id": "58bf3bb6e41b0738bf0000ce",
+    "user_id": "58bf3b87e41b0738bf0000cc",
+    "first_name": "Loren",
+    "last_name": "Dimallo"
+  },
+  "employers": [
+    {
+      "employer": {
+        "id": "58bf321ee41b0738bf00000c",
+        "legal_name": "BriCol",
+        "phone": "(783) 786-7857"
+      },
+      "broker": {
+        "id": "58c938e2e41b071e74000085",
+        "legal_name": "WilliamBroker",
```

Comment:
* Will add the "ridp_verified" field to this response and set it to TRUE.
* I am using the broker_agency_profile's legal name. Will add `employer_profile.active_broker.full_name` to the response.